test_data/yast/firstboot/yast2_firstboot_custom.yaml

```diff
@@ -0,0 +1,5 @@
+---
+custom_control_file: yast2/firstboot/firstboot_custom.xml
+clients:
+    - firstboot_welcome
```

Comment:
Here I chose to enable only hostname, but it could also make sense to enable the same modules + hostname. Please share your thoughts.

Reply:
as far as I remember the idea was to extend what was already there, so minimum would be to have the defaults + 1 (not only isolate this one because of the bug) and then we can extend this custom enabling most of them in some working combination in a follow-up ticket.

Reply:
done